geometry_msgs/PoseStamped pose
std_msgs/String new_frame
---
geometry_msgs/PoseStamped data
